app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -150,13 +151,33 @@ object MarkdownHelper {
                 }
             })
             .build()
+
+        noImageMarkwon = Markwon.builder(context)
+            // email urls interfere with lemmy links
+            .usePlugin(LinkifyPlugin.create(Linkify.WEB_URLS))
+            .usePlugin(LemmyLinkPlugin())
+            .usePlugin(StrikethroughPlugin.create())
+            .usePlugin(TablePlugin.create(context))
+            // use TableAwareLinkMovementMethod to handle clicks inside tables,
+            // wraps LinkMovementMethod internally
+            .usePlugin(MovementMethodPlugin.create(TableAwareMovementMethod.create()))
+            .usePlugin(HtmlPlugin.create()) // TODO - does this support <img ...> ? if so, we need to remove it
```

Comment:
should this also be removed from the no image markwon ?

Reply:
Markwon's html plugin does support images, but rather than disable it entirely, we can disable just the image tag support. Have a look here: https://noties.io/Markwon/docs/v4/html/#predefined-taghandlers

Reply:
updated 

```kotlin
            .usePlugin(HtmlPlugin.create { plugin -> plugin.addHandler(TagHandlerNoOp.create("img")) })
```